### PR TITLE
ENG-1162 add readme to categorize_notify

### PR DESCRIPTION
**Pull request by pashafateev**

### CODE REVIEW

File: categorize_notify/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Features
+- **Real-time Email Monitoring**: Continuously checks for new emails in the Gmail inbox.
+- **Automated Categorization**: Uses ChatGPT to analyze and categorize emails.
+- **Slack Notifications**: Sends categorized email content to the appropriate Slack channel.
+- **Email Labeling**: Adds labels to processed emails in Gmail for easy tracking.
+
+## Benefits
+- **Ease of Use:** Demonstrates how easy it is to connect multiple integrations into a cohesive workflow.
+- **Low Complexity:** The workflow is just over 100 lines of code.
```
Comment (daabr) [inline categorize_notify/README.md:12]: Some people may consider "100 lines of code" as a lot.

Actually, if you ignore empty lines, closing parens/brackets, etc. there's very little code left.

I'd rephrase this to "just a few lines of code" or something like that.

File: categorize_notify/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- **Free and Open Source:** Available for use or modification to fit specific use cases. 
+
+## How It Works
```
Comment (daabr) [inline categorize_notify/README.md:15]: This section feels repetitive... Maybe merge it with "Features"? If so, I'm not sure which header is better.

Comment (pashafateev) [reply categorize_notify/README.md:9]: Done

File: categorize_notify/README.md (hunk 1 above)
Comment (pashafateev) [reply categorize_notify/README.md]: I agree. Done

**Review by daabr — APPROVED** [review]
